CONTRIBUTING.md

```diff
@@ -4,11 +4,11 @@ Kaizen Design System follows a federated, open-source model. All teams contribut
 
 ### Quality and reviews
 
-Every component needs:
+Every contribution **must** have:
 
 - A design review from a designer
-- Basic level of unit tests
```

Comment:
Perhaps a new section:

```
Every contribution **should strive to** have:
- Basic level of unit tests
- New or updated Storybook stories
```

So, we're saying that it's releasable without, but we'd really like you to add them (sometimes people simply forgot to write tests, so it's good to have a reminder for those who have the time)

Reply:
Good call, will add.